packages/expo-cli/src/commands/eject/Eject.js

```diff
@@ -227,6 +227,13 @@ from \`babel-preset-expo\` to \`babel-preset-react-native-stage-0/decorator-supp
     pkgJson.scripts.ios = 'react-native run-ios';
     pkgJson.scripts.android = 'react-native run-android';
 
+    const { sdkVersion } = exp;
+    const versions = await Versions.versionsAsync();
+    const reactNativeVersion = versions['sdkVersions'][sdkVersion]['facebookReactNativeVersion'];
+
+    pkgJson.dependencies['@babel/runtime'] = 'latest';
```

Comment:
This is risky, because it always pulls in the latest version of the package, so any new release of `@babel/runtime` might break it.

As far as I know, this issue also only affected react-native 0.57.1 ([0.57.2 added `@babel/runtime` in RN's dependencies](https://github.com/facebook/react-native/blob/4a440f40189ea54bd64f7a03bd05bf26619ad320/package.json#L152)).

Because of this, I think it would be best to use a specific version of `@babel/runtime` and limit the fix to only those affected versions (only SDK 31 and SDK 32 used the 0.57.1 fork, SDK 33 will use 0.59):
```suggestion
    if (semver.satisfies(sdkVersion, '31 - 32')) {
      pkgJson.dependencies['@babel/runtime'] = '^7.0.0';
    }
```

Reply:
Good call.